### PR TITLE
Add DeFi Oracle Meta (138 Mainnet and 2138 Testnet)

### PR DESCRIPTION
**Pull request by defi-oracle**

### CODE REVIEW

File: _data/chains/eip155-138.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+   "name": "Defi Oracle Meta Mainnet",
```
Comment (Bjonesca30) [inline _data/chains/eip155-138.json:2]: []()@
```suggestion
   "name": "Defi Oracle Meta Mainnet",```suggestion
   "name": "Defi Oracle Meta Mainnet",**$@9**
```

```

**Review by Bjonesca30 — APPROVED** [review]

**Review by defi-oracle — COMMENTED** [review]
cool

**Comment by robledo6** [comment]
Aveeer

**Review by ligi — CHANGES_REQUESTED** [review]
Do not touch files outside _data

**Review by robledo6 — APPROVED** [review]

**Comment by Mohito62** [comment]
Hi

**Comment by defi-oracle** [comment]
Hello.

**Pull request closed without merging**